recipes/boost/all/conanfile.py

```diff
@@ -1045,23 +1045,23 @@ def _build_cross_flags(self):
 
     @property
     def _ar(self):
-        if "AR" in os.environ:
+        if os.environ.get("AR"):
```

Comment:
Use `tools.get_env` instead?

Reply:
It looks like `tools.get_env` (aka `conans.util.env_reader.get_env`) doesn't really do anything when the default is `None` or a `str` as it is in this case, it seems like the purpose is to coerce the type when `default` is a `bool`, `int`, `float`, or `list`.